core/sdk-core/src/main/java/software/amazon/awssdk/core/SdkPlugin.java

```diff
@@ -23,6 +24,7 @@
  * A plugin modifies a client's configuration when the client is created or at request execution
  * time.
  */
+@SdkPreviewApi
 @SdkPublicApi
```

Comment:
nit: public and preview. i guess it's ok.

Reply:
It's actually needed, for classes you have to annotate them with any of public, protected or internal. IOW, preview does not replace public.